src/activities/gletters/resource/default-en.json

```diff
@@ -7,6 +7,16 @@
             "e",
             "i",
             "o",
+            "u",
```

Comment:
I understand what you want to do and it makes sense. But I don't like to to it by extendind the dataset this way. We have a property "sublevels" that can be used in the level. I tested it and it does not do the trick to enlarge the dataset up to this number. If you are interested, I propose you to change the code to allow this.